view/frontend/templates/js/replacejs.phtml

```diff
@@ -35,6 +35,62 @@ ob_start();
     // in payment method renderer, vendor/boltpay/bolt-magento2/view/frontend/web/js/view/payment/method-renderer/boltpay.js
     window.boltConfig = <?php echo $block->getSettings(); ?>;
 
+    var trackCallbacks = {
+        onCheckoutStart: function() {
+            try {
+                <?php echo $trackCallbacks['checkout_start']; ?>
```

Comment:
Question: Is this a separate function also called trackCallbacks?

Reply:
tweaked a code bit. this is PHP variable with which we construct JS variable named trackCallbacks here